frontend/src/components/projects/ProjectsListContent.js

```diff
@@ -0,0 +1,175 @@
+import React, {useRef} from "react";
+import {connect} from "react-redux";
+import {Link} from "react-router-dom";
+import {Button, Tag} from "antd";
+
+import AppPageHeader from "../AppPageHeader";
+import PageContent from "../PageContent";
+import PaginatedTable from "../PaginatedTable";
+import AddButton from "../AddButton";
+import ExportButton from "../ExportButton";
+
+import api, {withToken}  from "../../utils/api"
+
+import {listTable, setFilter, setFilterOption, clearFilters, setSortBy} from "../../modules/projects/actions";
+import {actionsToButtonList} from "../../utils/templateActions";
+import {withContainer, withIndividual} from "../../utils/withItem";
+import {PROJECT_FILTERS} from "../filters/descriptions";
+import getFilterProps from "../filters/getFilterProps";
+import getNFilters from "../filters/getNFilters";
+import FiltersWarning from "../filters/FiltersWarning";
+import ProjectsFilters from "./ProjectsFilters";
+import mergedListQueryParams from "../../utils/mergedListQueryParams";
+
+const getTableColumns = () => [
+    {
+      title: "ID",
+      dataIndex: "id",
+      sorter: true,
+      width: 80,
+      render: (id, project) =>
+        <Link to={`/project/${project.id}`}>
+          <div>{id}</div>
+        </Link>,
+    },
+    {
+      title: "Name",
+      dataIndex: "name",
+      sorter: true,
+      width: 80,
+      render: (name, project) =>
+        <Link to={`/project/${project.id}`}>
+          <div>{name}</div>
+        </Link>,
+    },
+    {
+      title: "Principal Investigator",
+      dataIndex: "principal_investigator",
+      sorter: true,
+      width: 70,
+    },
+    {
+      title: "Requestor Name",
+      dataIndex: "requestor_name",
+      sorter: true,
+      align: "right",
+      className: "table-column-numbers",
+      width: 100,
+    },
+    {
+      title: "Requestor Email",
+      dataIndex: "requestor_email",
+      align: "right",
+      className: "table-column-numbers",
+      width: 115,
+    },
+    {
+      title: "Targeted End Date",
+      dataIndex: "targeted_end_date",
+      sorter: true,
+      align: "right",
+      className: "table-column-numbers",
+      width: 115,
+    },
+    {
+      title: "Status",
+      dataIndex: "status",
+      sorter: true,
+      align: "right",
+      className: "table-column-numbers",
+      width: 115,
+    },
+    {
+      title: "Comments",
```

Comment:
We never (or almost never) add the 'Comments' fields in the columns of a list of items. In the event where we would have a very lengthy comment, the table would become ugly :P also the list should not display all the attributes of a Model, but the most significant ones